src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -2485,7 +2485,7 @@
   "com.minecolonies.coremod.gui.townhall.stats.items_delivered": "Delivered Items: %d",
   "com.minecolonies.coremod.gui.townhall.stats.food_served": "Served Food: %d",
   "com.minecolonies.coremod.gui.townHall.map": "Town Map",
-  "com.minecolonies.coremod.townhall.map.warning": "Drop-off a normal scale Minecraft map in the Town Hall inventory first to unlock the map.",
+  "com.minecolonies.coremod.townhall.map.warning": "Drop-off a Minecraft map(s) in the Town Hall inventory first to unlock the map.",
```

Comment:
This is to make sure they only drop the small ones off, not the big ones

Reply:
Read the PR doc :D

Reply:
Yeah, but that was gameplay intentional to have maps only of a fixed size in there =D

Reply:
reminder